### DISCUSSION TITLE
DELETE 删除后面不能加子句吗?

### DISCUSSION THREAD

**Discussion by Hby118** [apache/doris#21312] [Ideas]
场景是这样的,有两个结构一样的AB表

如何删除A表中再B中一定条件存在的数据
比如
DELETE FROM A
WHERE ID IN (SELECT ID FROM B WHERE ... )

**Comment by yagagagaga** [comment]
NOT ALLOWED.
You can use `delete using` clause in Doris 2.0.

**Reply by Hby118** [reply]
好吧,我就看到手册有2.0 语法介绍了,估计我这没这么快能用上2.0

那有什么别的办法能实现类似的效果吗,目前用的办法是遍历每一条记录,然后逐条删除,效率太低了.

**Reply by yagagagaga** [reply]
You can select and insert to another table.

**Reply by nospeak12345** [reply]
how to upgrade my doris from PALO 1.1.0 to 2.0 safely?

**Reply by ixzc** [reply]
refer this ： https://doris.apache.org/zh-CN/docs/dev/admin-manual/cluster-management/upgrade
